packages/primitives/src/svg/index.js

```diff
@@ -80,7 +80,7 @@ export const Stop = ( props ) => createElement( 'stop', props );
 
 /**
  *
- * @param {SVGProps} props
+ * @param {Object}   props
  * @param {string}   props.className Class name
  * @param {boolean}  props.isPressed Is the SVG currently pressed?
  * @param {import('react').ComponentPropsWithoutRef<'svg'>} props.props Other props will be passed thru to svg element
```

Comment:
This should be something like:

```suggestion
 * @param { { isPressed?: boolean } & import('react').ComponentPropsWithoutRef<'svg'>} props Other props will be passed thru to svg element
```

Note: `props.props` is _not_ where the rest props go. I've seen this repeated in a few places. The generated documentation will suffer with a change like this, but we can help compensate with a good description.